### ISSUE TITLE
Fix `parity/readme.md`: it cites a strange paper

### ISSUE THREAD

**Issue by 5nizza** [SYNTCOMP/benchmarks#20]
Either explain or remove or find a new home for the following line from `parity/readme.md`:
```
The paper on [A symmetric protocol to establish service level agreements](https://lmcs.episciences.org/6812)
```
It is unclear why this paper is there (I assume because some of the benchmarks are from that paper, then move that reference to a benchmark folder, may be?)

**Comment by gaperez64** [comment]
Yes, it's also mentioned in `parity/mucalc_mc/readme.md` where it is better explained what the context is. I will just remove it from that readme as it might indeed be confusing.
